assets/kube-state-metrics/deployment.yaml

```diff
@@ -38,8 +38,6 @@ spec:
         - --telemetry-port=8082
         - |
           --metric-denylist=
-          ^kube_secret_labels$,
-          ^kube_.+_annotations$
```

Comment:
cc @rexagod, apparently the comma was omitted during https://github.com/openshift/cluster-monitoring-operator/commit/bc1174a6f7746950a11bc405bc825a132c4957a7, not sure if this was intentional.